src/petals/server/memory_cache.py

```diff
@@ -12,24 +12,26 @@
 import time
 from typing import AsyncContextManager, Dict, Optional, Sequence
 
-import hivemind
+import async_timeout
 import torch
-from hivemind.utils import TensorDescriptor, get_logger
+from hivemind.utils import TensorDescriptor, enter_asynchronously, get_logger
```

Comment:
```suggestion
from hivemind.utils import TensorDescriptor, get_logger
```

No need for this (see below)